.github/workflows/release.yml

```diff
@@ -0,0 +1,33 @@
+name: Release
+
+on:
+  pull_request:
+    branches:
+      - master
+    types: [closed]
```

Comment:
Q: Genügt das als Loop-Detection?

Reply:
Müsste in Kombination mit L11 reichen.

Reply:
Was ist L11

Reply:
Line 11: `if: github.event.pull_request.merged == true`